src/actor.rs

```diff
@@ -610,7 +617,9 @@ mod test {
 
         // 7 elders and validations
         for i in 0..7 {
```

Comment:
This wants updated to be 5 now by default I believe

Reply:
That's just a test so it should be okay?